static/src/js/components/Tour/SearchTour.jsx

```diff
@@ -103,6 +103,7 @@ const SearchTour = () => {
       stepIndex={stepIndex}
       continuous
       callback={handleJoyrideCallback}
+      disableScrollParentFix
```

Comment:
https://docs.react-joyride.com/props - this was to fix an issue for some cases on joyride in our case with the wrapping-container this was adding a style that messed up the overflows we added in SCSS
it was adding `overflow: initial` to the parent element of the Search Tour